ingredient/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+from django.db.models.deletion import CASCADE
+
+from recipe.models import Recipe
+from user.models import User
+
+class MainCategoryIngredient(models.Model):
+    name = models.CharField(max_length=10)
+
+    class Meta:
+        db_table = 'main_categories_ingredients'
+
+class SubCategoryIngredient(models.Model):
+    name = models.CharField(max_length=10)
+    main = models.ForeignKey(MainCategoryIngredient, on_delete=models.CASCADE)
+
+    class Meta:
+        db_table = 'sub_categories_ingredients'
```

Comment:
아래와 같이 수정하시면 좋을 것 같아요~
1. `Category`라는 의미가 명확하도록 class 명을 같이 수정해주시면 어떨까요? (table이름도 동일하게요)
2. `created_at` column을 추가해주시면 좋을 것 같습니다.
3. foreignkey로 참조하고 있는 컬럼 명을 `main_category_id`와 같이 id로 수정해주세요~ 실제로 저장되는 데이터는 main_category 객체가 아니라 id값 입니다.

```suggestion
class IngredientMainCategory(models.Model):
    name = models.CharField(max_length=10)
    created_at = models.DateTimeField(auto_now_add = True)
    
    class Meta:
        db_table = 'main_categories_ingredients'

class IngredientSubCategory(models.Model):
    name = models.CharField(max_length=10)
    main_category_id = models.ForeignKey(MainCategoryIngredient, on_delete=models.CASCADE)
    
    class Meta:
        db_table = 'ingredient_sub_categories'
```

Reply:
수정완료했습니다.